app/src/main/java/org/fossasia/openevent/general/auth/LoginFragment.kt

```diff
@@ -81,14 +81,14 @@ class LoginFragment : Fragment() {
         loginViewModel.progress
             .nonNull()
             .observe(this, Observer {
-                handleProgressBar(it)
+                handleProgressBar(it, progressDialog)
```

Comment:
Please declare progressDialog as `private lateinit var` before onCreateView and initialize it in onCreateView. So that there is no need to pass progressDialog.

Reply:
As @iamareebjamal suggested, no need to create function